build/pause/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.10
```

Comment:
Can you please update the following line as well?
https://github.com/kubernetes/kubernetes/blob/master/build/pause/Makefile#L20

You will need to comment out the following as well for the CI jobs to be green:
https://github.com/kubernetes/kubernetes/blob/master/build/dependencies.yaml#L192-L196

Reply:
updated.